pkg/services/file_uploads.go

```diff
@@ -21,3 +26,18 @@ func NewCreateFileUploadURL(config Config, s3client upload.S3Client) handlers.Cr
 		return url, nil
 	}
 }
+
+// NewCreateUploadedFile returns a function that saves the metadata of an uploaded file
+func NewCreateUploadedFile(config Config, create createFunc) handlers.CreateUploadedFile {
```

Comment:
Hrm... Doesn't this create an import cycle of `handers` depending on `services` depending on `handlers`? Or do we get around that because it doesn't get woven together until the `server/routes.go` is invoked?

Reply:
Yah, I think the pattern that has been used in the `services` package is to just declare the functional signature, without using an alias (from another package), keeping the dependencies lighter-weight.

Reply:
Ah yeah, good catch! I meant to fix this up. The massive function signature in this services pattern was making it hard to read and reason about in my emacs as I was laying this all out.

I feel like there's probably an opportunity to tidy things up, as when I was first reading this pattern it was giving me Java vibes, but for now I'll fix this to be the same as the other `services`.